topics/proteomics/tutorials/proteogenomics-novel-peptide-analysis/tutorial.md

```diff
@@ -195,123 +203,121 @@ Once this step is completed, a tabular output containing novel proteoforms are d
+>      > 
+>      > ![MVP](../../images/Open_MVP.png){:width="20%"}
+>      > 
+>      > This is how it will look.
+>      > 
+>      > ![Overview](../../images/Peptide_overview.png){:width="60%"}
+>      > 
+> 2. Click on **Load from Galaxy** to open the list of peptides you would like to view.
+>      > 
+>      > ![load from galaxy](../../images/load_from_Galaxy.png){:width="80%"}
+>      > 
+>      > This will open a dropdown list. Select the novel peptides from there.
+>      > 
+>      > 
+>      > ![dropdown](../../images/DropDown_novel.png){:width="50%"}
+>      > 
+> 3. Select **Novel Peptides** from the right hand side.
+>      > 
+>      > ![novel peptides view](../../images/novel_peptides_view.png){:width="70%"}
+>      > 
+> 4. Select any peptide, For eg: `DGDLENPVLYSGAV`, and then click on **PSMs for Selected Peptide**.
+>      > 
+>      > ![select_pep](../../images/Filtering_novelpeptides.png){:width="70%"}
+>      > 
+> 5. If you scroll down, the PSM associated with the peptide will be displayed. By clicking on the PSM, the *Lorikeet*
 > values will be shown. The Lorikeet visualization is interactive, i.e the user can change the values or select any
 > parameter and click on Update button to view these changes.
->
-> ![PSM](../../images/Peptide-Protein_Viewer.png){:width="90%"}
->
-> ![lorikeet](../../images/Lorikeet_Viewer.png){:width="70%"}
->
->
-> 6) For a Protein centric view, click on **View in Protein** , it will open up all the proteins associate with the
+>      > 
+>      > ![PSM](../../images/Peptide-Protein_Viewer.png){:width="90%"}
+>      > 
+>      > ![lorikeet](../../images/Lorikeet_Viewer.png){:width="70%"}
+>      > 
+>      > 
+> 6. For a Protein centric view, click on **View in Protein** , it will open up all the proteins associate with the
 > peptides. For eg: Select the `DGDLENPVLYSGAV` peptide and click on the first protein. The chromosome location
 > of the peptide will be displayed.
->
-> ![view in protein](../../images/Selecting_peptide.png){:width="70%"}
->
-> Once you click on protein it will show the list of proteins the belongs to the peptides.
->
-> ![select protein](../../images/opening_IGV.png){:width="60%"}
->
-> Once you select the protein that you want to visualize you can click on the protein view.
->
-> ![protein view](../../images/ProteinViewer.png){:width="80%"}
->
->
-> 7) Clicking on the arrow marks will open up the IGV(js) visualization tool, where-in the genomic localization of the
+>      > 
+>      > ![view in protein](../../images/Selecting_peptide.png){:width="70%"}
+>      > 
+>      > Once you click on protein it will show the list of proteins the belongs to the peptides.
+>      > 
+>      > ![select protein](../../images/opening_IGV.png){:width="60%"}
+>      > 
+>      > Once you select the protein that you want to visualize you can click on the protein view.
+>      > 
+>      > ![protein view](../../images/ProteinViewer.png){:width="80%"}
+>      > 
```

Comment:
these extra `>` are not necessary (and causing the lint failures). As long as you line everything up with the C from "7. Clicking" above the indentations should be correct and not break the numbering